### PR TITLE
🦤 add gitlab pat 🦤

### LINKED ISSUE #65: 🐛 [bug] - Passwords containing %,#,// break a Jenkins pipeline
During testing ....

At the end of https://rht-labs.com/tech-exercise/#/2-attack-of-the-pipelines/3a-jenkins , this bit of the pipeline fails:

```
git clone https://${GIT_CREDS}@${ARGOCD_CONFIG_REPO} config-repo
```
with:
```
fatal: unable to access 'https://gitlab-ce.apps.tl500-1.ls-eu.ole.redhat.com/fulanitos/tech-exercise.git/': Port number ended with 'X'
```
that is likely caused by my password containing X followed by two :.

### PR DESCRIPTION
**Pull request by eformat**
add gitlab pat
fixes #65 

take a look see ! this needs to merge along with enablement repo which i will link here

### CODE REVIEW

**Review by pabrahamsson — APPROVED** [review]
:heart_eyes_cat: :heart_eyes_cat: :heart_eyes_cat: 
LGTM

File: docs/1-the-manual-menace/3-ubiquitous-journey.md [hunk 1]
```diff
@@ -37,13 +37,49 @@ All of these traits lead to one outcome - the ability to build and release quali
 5. On the new view, use `tech-exercise` as Project Name, select **Internal** for Visibility level, then hit Create project. Make sure the project is in the group you created previously and not the username's.
 ![gitlab-new-project](images/gitlab-new-project-2.png)
 
-6. Let's push our code to the GitLab server. Back in your CodeReady Workspace, open a terminal if you have not got one open.
+6. We are going to create a gitlab personal access token (PAT). The token is a more secure and reliable method for accessing gitlab from our scripts later on. Note, that for reference's sake, you can also generate a PAT in gitlab under User > Settings > Access Tokens in the Web UI. We use a helper script here to help automate that process. To generate the token, open a terminal if you have not got one open and run the following commands
```
Comment (ckavili) [inline docs/1-the-manual-menace/3-ubiquitous-journey.md:40]:
```suggestion
6. We are going to create a GitLab Personal Access Token (PAT). The token is a more secure and reliable method for accessing GitLab from our scripts later on. Note, that for reference's sake, you can also generate a PAT in GitLab under User > Settings > Access Tokens in the Web UI. We use a helper script here to help automate that process. To generate the token, open a terminal if you have not got one open and run the following commands.
```

File: docs/1-the-manual-menace/3-ubiquitous-journey.md [hunk 2]
```diff
@@ -37,13 +37,49 @@ All of these traits lead to one outcome - the ability to build and release quali
...
+
+    Export your gitlab username
```
Comment (ckavili) [inline docs/1-the-manual-menace/3-ubiquitous-journey.md:42]:
```suggestion
    Export your GitLab username
```

File: docs/1-the-manual-menace/3-ubiquitous-journey.md [hunk 3]
```diff
@@ -37,13 +37,49 @@ All of these traits lead to one outcome - the ability to build and release quali
...
+
+    ```bash
+    export GITLAB_USER=<YOUR_GITLAB_USER>
+    ```
+
+    Export your gitlab password
```
Comment (ckavili) [inline docs/1-the-manual-menace/3-ubiquitous-journey.md:48]:
```suggestion
    Export your GitLab password.
```

File: docs/1-the-manual-menace/3-ubiquitous-journey.md [hunk 4]
```diff
@@ -37,13 +37,49 @@ All of these traits lead to one outcome - the ability to build and release quali
...
+
+    ```bash
+    export GITLAB_PASSWORD=<YOUR_GITLAB_PASSWORD>
+    ```
+
+    <p class="tip">
+    ⛷️ <b>TIP</b> ⛷️ - If your password includes special characters, try putting it in single quotes. ie: <strong>'A8y?Rpm!9+A3B/KG'</strong>
+    </p>
+
+    Generate your gitlab PAT
```
Comment (ckavili) [inline docs/1-the-manual-menace/3-ubiquitous-journey.md:58]:
```suggestion
    Generate your GitLab PAT.
```

**Pull request merged**